src/utils/statistics.ts

```diff
@@ -30,12 +30,17 @@ const Statistics = {
 
         tokens.wastedSeconds += Statistics.timeTags.parseElapsed ( tag );
 
-      } else if ( !disabledTokens.est && !disabledEst && ( prefix === 'e' || ( prefix >= '0' && prefix <= '9' ) ) ) { // Maybe @est(1h20m) or @1h20m
+      } else if ( !disabledTokens.est && ( prefix === 'e' || ( prefix >= '0' && prefix <= '9' ) ) ) { // Maybe @est(1h20m) or @1h20m
 
-        tokens.estSeconds += Statistics.timeTags.parseEstimate ( tag );
+        // always add to total
+        tokens.estTotalSeconds += Statistics.timeTags.parseEstimate ( tag );
 
-      }
+        // only add if not disabled
+        if(!disabledEst) {
```

Comment:
I'd write it like so: `if ( !disabledEst ) {`

Reply:
See newest commit